audio/tauonmb/tauonmb.SlackBuild

```diff
@@ -76,7 +76,9 @@ mkdir -p lib
 mv libphazor.so lib/libphazor.so
 
 # Compile translator
-python3 compile-translations.py install --root=$PKG
+python3 compile-translations.py
```

Comment:
Was the removal of "install" and "--root=$PKG" intentional?

Reply:
Hello Mr Workman,
It doesn't really target the $PKG, instead it makes .mo files under locale itself locally anyway, which I move them after converting .po and removing them afterward;
It's not necessary to specify the target there